docs/reference/templatetags.rst

```diff
@@ -112,12 +123,22 @@ Example::
 Note that the `Django "sites" framework <https://docs.djangoproject.com/en/dev/ref/contrib/sites/>`_ *is* required and
 ``SITE_ID`` *must* be set in ``settings.py`` for this (not to mention other aspects of django CMS) to work correctly.
 
+..  warning::
+
+    Placeholders may be added inside the template's ``<head>``. There are very few cases where you
```

Comment:
I can't imagine one, a specific template tag might be better? @vxsx AFAIK we wouldn't it be a problem to place cms_toolbar in between the body and header?

Reply:
template tag is _always_ better, and placeholders in head will never work since 
you cannot put `{% cms_toolbar %}` in the head, since it produces toolbar markup 
page extension or custom template tag is the solution, not a placeholder

Reply:
Actually I tested with `{% cms_toolbar %}` and placeholder template tag in the `<head>` - it works, but not as expected (they appear in the body in the DOM anyway).

Reply:
that depends on the browser

Reply:
I suggest to add a note saying this is not possible.